config/default/kustomization.yaml

```diff
@@ -1,17 +1,3 @@
-# Adds namespace to all resources.
-namespace: capi-migration-system
-
-# Value of this field is prepended to the
-# names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
-# Note that it should also match with the prefix (text before '-') of the namespace
-# field above.
-namePrefix: capi-migration-
```

Comment:
I needed to remove name prefix from here because it wasn't added overlays and I don't want intermediate prefixes because that makes resource names really long.